### ISSUE TITLE
Select-only combobox example: selection doesn't follow focus

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#1624]
The [Select-Only Combobox Example](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html) presents a single-select widget; multiple options cannot be selected.  `aria-activedescendant` is used to manage focus among the options of the listbox pop-up, but `aria-selected` is explicitly set to `"false"` on each option and only set to `"true"` once an action is executed to confirm selection and close the pop-up ((such as a mouse click or press of Enter).

This differs from the standard behaviour of an HTML select, whereby selection follows focus.  As this is a single-select widget, the selected state of any individual option is irrelevant.  It also creates additional speech and braille verbosity in those screen readers which convey something for `aria-selected="false"`.

Could it be clarified why `aria-activedescendant` alone isn't sufficient for this control?

**Comment by carmacleod** [comment]
Good question @jscholes.

In case it's useful to know, the [Editable Combobox With List Autocomplete Example](https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-list.html) only uses `aria-selected="true"`, and removes the `aria-selected` attribute when an option is not selected. However, maybe even that is not necessary, because this can result in hearing "selected" for every down arrow when navigating through the options.

**Comment by jscholes** [comment]
@carmacleod Exactly.  I may be wrong, but I think the intent is to indicate which option has the logical focus inside the listbox, and I think `aria-activedescendant` plus the corresponding visual styling would fulfill that without `aria-selected` being required.

**Comment by carmacleod** [comment]
@jscholes 
> aria-activedescendant plus the corresponding visual styling would fulfill that without aria-selected being required.

Agreed. Or if the component is using roving tabindex, then `tabindex="0"` plus the corresponding visual styling would also work.

The only question I have is around things like the [Editable Combobox without Autocomplete Example](https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-none.html), or any combobox that does not filter the values. In this case, the user can press Esc (or click elsewhere) to close the listbox without changing the combobox selection. Not sure if `aria-selected="true"` might be useful there somehow to indicate the combobox's current value the next time the user navigates through the listbox?

**Comment by carmacleod** [comment]
Just remembered that the [ARIA spec for option](https://w3c.github.io/aria/#option) describes the conditions for user agents to provide an implicit value for aria-selected:
> In certain conditions, a user agent MAY provide an implicit value for aria-selected for each option in a listbox, and if it does, the user agent MUST ensure the following conditions are met before providing an implicit value:
>
> - The value of aria-multiselectable on the listbox is false or undefined.
> - None of the option elements in the listbox have an explicitly declared value for aria-selected or aria-checked.
>
> If a user agent provides an implicit aria-selected value for an option, the value SHOULD be true if the option has DOM focus or the listbox has DOM focus and the option is referenced by aria-activedescendant. Otherwise, if a user agent provides an implicit aria-selected value for an option, the value SHOULD be false.

So possibly we should consider removing aria-selected from all of these combobox examples, and just let the user agents compute an implicit value. (unless the user agents don't all do this correctly?)
